tests/storage_test.go

```diff
@@ -73,8 +73,8 @@ var _ = Describe("Storage", func() {
 				Expect(err).ToNot(HaveOccurred())
 				expecter.Close()
 			},
-				table.Entry("with Disk PVC", tests.NewRandomVMIWithPVC),
-				table.Entry("with CDRom PVC", tests.NewRandomVMIWithCDRom),
+				table.Entry("[test_id:3130]with Disk PVC", tests.NewRandomVMIWithPVC),
```

Comment:
not that it matters for getting the PRs approved, but is there a reason why we never have a space after the `]`?